stack/indexer/deb/debian/preinst

```diff
@@ -7,10 +7,11 @@
 # License (version 2) as published by the FSF - Free Software
 # Foundation.
 
-set -e 
+set -e
 
 export NAME=wazuh-indexer
 export CONFIG_DIR="/etc/${NAME}"
+export BACKUP_DIR=/usr/share/backup
```

Comment:
If possible create the backup inside the installation directory.

Reply:
Done.

Reply:
I noticed the `INSTALLATION_DIR` directory is removed in the upgrade, so the backup directory can not be inside that directory. I propose: `${CONFIG_DIR}/backup`
https://github.com/wazuh/wazuh-packages/pull/1998/commits/8bbf5f81025723e511fc7cc42d82573d55825823